### DISCUSSION TITLE
The local adapter does not support an important Low Energy controller state to support peripheral mode

### DISCUSSION THREAD

**Discussion by ShinoIro** [nefarius/DsHidMini#268] [Q&A]
The local adapter does not support an important Low Energy controller state to support peripheral mode on Event Viewer
is this mean that i'm out of luck? 
the ds3 detected as fake tho, but the warning wasn't the one that mentioned on Q&A, so maybe... there's a chance?
Here's the image
![image](https://user-images.githubusercontent.com/66824646/209400237-d6575094-febb-4eb0-82e5-58c4a339c827.png)
i used the built in bluetooth to connect, i;m guessing that's the problem..

**Comment by nefarius** [comment]
This message has absolutely nothing to do with BthPS3 or its operation. BthPS3 has its own event logging source.
